src/clear.js

```diff
@@ -0,0 +1,24 @@
+import task from './list';
+
+export const clear = (e) => {
+  const clicked = e.target.closest('.check');
+  if (!clicked) return;
+  const index = parseInt(clicked.getAttribute('data-clear'), 10);
```

Comment:
Well done @anasbassi, good job, and great functionalities you have. I love how you keep your code short and dry, but I will like to suggest the following changes in accordance with KISS, DRY and YAGNI principles...

could you please tell me what 10 is doing in your const index?

Reply:
hi @Lizdev-05 
it is called the radix, it is an integer between 2 and 36  (the base in mathematical numeral systems) of the string. If outside this range, the function will always return NaN. If 0 or not provided, here is a link to an MDN post(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt) for more clarification.